services/dynamic-scheduler/src/simcore_service_dynamic_scheduler/api/rpc/_services.py

```diff
@@ -37,6 +38,7 @@ async def run_dynamic_service(
     response: NodeGet | DynamicServiceGet = (
         await director_v2_client.run_dynamic_service(dynamic_service_start)
     )
+    await set_request_as_running(app, dynamic_service_start)
```

Comment:
minor: for me would make more sense to either have
set_request_as_started(app, dynamic_service_start)
or
set_request_as_running(app, dynamic_service_running)